README.md

```diff
@@ -128,22 +134,22 @@ For this Demo, both the necessary Condition and Action smart contracts have alre
 
 #### Gelato Providers & Executors
 
-Gelato **automates `Task` execution** by incentivising a **network of relay servers** to **execute** them **for your Dapp Users**.
+Gelato consists of a **network of relay servers** that are being incentivized to **execute** certain `Tasks` **for the Users of your Dapp**.
 
-These `Executors` need to be paid, in order to operate their automation infrastructure.
+These `Executors` need to be paid, in order to operate their automation infrastructure and get compensated for submitting transactions.
 
 In Gelato **`Providers` pay `Executors`** to **automate** `Tasks`.
 
 There are two kinds of `Providers`:
 
-1. `Provider`: pays `Executors` on behalf of his/her Dapp Users
-2. `Self-Provider`: a Dapp User that pays `Executors` himself/herself
+1. `External Providers`, which pay `Executors` on behalf of his/her Dapp Users. These are dapp developers that want to provide users with a great UX by removing the necessity of depositing ETH on gelato from them.
```

Comment:
Call them `Gelato Providers`

Reply:
Might be a bit weird that Self Providers are not Gelato Providers. We need a distinction between self and not self rather than Gelato, not gelato